packages/form-js-editor/src/render/components/properties-panel/entries/ValuesSourceEntries.js

```diff
@@ -0,0 +1,219 @@
+import { get, isUndefined, without } from 'min-dash';
+
+import { SelectEntry, isSelectEntryEdited, TextFieldEntry } from '@bpmn-io/properties-panel';
+import ValueEntry from './ValueEntry';
+
+import { arrayAdd } from '../Util';
+import { useService } from '../../../hooks';
+
+export function ValuesSourceSelectorEntry(props) {
+  const {
+    editField,
+    field
+  } = props;
+
+  return [ {
+    id: 'values-source',
+    component: ValuesSourceSelect,
+    isEdited: isSelectEntryEdited,
+    editField,
+    field,
+  } ];
+}
+
+export function InputKeyValuesSourceEntry(props) {
+  const {
+    editField,
+    field
+  } = props;
+
+  return [{
+    id: 'input-values-key',
+    component: InputValuesKey,
+    label: 'Input values key',
+    description: 'Define which input property to populate the values from',
+    isEdited: isSelectEntryEdited,
+    editField,
+    field,
+  }];
+}
+
+export function StaticValuesSourceEntry(props) {
+  const {
+    editField,
+    field,
+    id: idPrefix
+  } = props;
+
+  const {
+    values
+  } = field;
+
+  const addEntry = (e) => {
+
+    e.preventDefault();
```

Comment:
```suggestion
    e.stopPropagation();
```

Otherwise, it leads to this behavior

![Kapture 2022-07-07 at 13 14 31](https://user-images.githubusercontent.com/9433996/177760868-eb16ca59-90af-4a85-8787-ce67f66482b3.gif)